### PR TITLE
Update on gitignore and readme files. I deleted unnecessary folders and files from GitHub.

### PR DESCRIPTION
**Pull request by MiKessels**

### CODE REVIEW

**Comment by MiKessels** [comment]
Please leave some feedback if gitignore is strange or readme.md could be different.

File: README.md [hunk 1]
```diff
@@ -12,32 +15,39 @@ Clone or download the code from this GitHub. The stable version is the acceptanc
...
 
+## License
+
+This project is licensed under the MIT License - ~~see the LICENSE.md file for details.~~
+
+## Docs
+
+~~see the DOCS for details~~
+
 ## Changelog
```
Comment (cyrildewit) [inline README.md:46]: Are you familiar with keepachangelog.com? I also think that the changelog belongs in it's own file. See https://github.com/cyrildewit/eloquent-viewable/blob/master/CHANGELOG.md for example or see what other open source projects do.

**Review by cyrildewit — CHANGES_REQUESTED** [review]
See my comment about the changelog.

File: README.md (hunk 1 above)
Comment (MiKessels) [reply README.md:46]: Yep, you told me earlier. Didn't start using changelog yet. I will delete the changelog for now and create a different file for the changelog, like ' the docs'

**Review by cyrildewit — APPROVED** [review]

**Comment by cyrildewit** [comment]
@MiKessels approved. LGTM.

**Pull request merged**